features/desktop/notifications.feature

```diff
@@ -101,6 +101,26 @@ Feature: Notifications
     Then I should see "mentioned you in the post"
     And I should have 1 email delivery
 
+  Scenario: someone mentioned me in a comment
+    Given a user with email "bob@bob.bob" is connected with "alice@alice.alice"
+    And "alice@alice.alice" has a public post with text "check this out!"
+    And "bob@bob.bob" has commented mentioning "alice@alice.alice" on "check this out!"
+    When I sign in as "alice@alice.alice"
+    And I follow "Notifications" in the header
+    Then the notification dropdown should be visible
+    Then I should see "mentioned you in the comment"
+    And I should have 1 email delivery
+
+  Scenario: I mark a mention in comment notification as read
```

Comment:
I agree with @SuperTux88 here. We use cukes to test the combination of front- and backend. And when doing that at least I also have the implementation in mind to know, which cases I have to go through.

You could use different “mark read/unread” cukes for different notification types. You could also use different “mark read/unread” cukes for contact / non-contact actors. Or users on the same pod vs. users on different pods. Or for posts with photos and posts without photos. Or... (I hope you get my point ;))

In this case there is exactly one thing we have to test: Do we send the correct request to the correct route when clicking the button. And the requests all look the same for the different notification types. And the buttons also look the same.

I still remember the times when our cucumber tests exceeded the timeout of 50 minutes. Now they need about 15 minutes but we have a few problems with randomly failing tests. Thus I'd still like to keep the number of cukes rather small. Let's keep using cukes to test the different combinations of front- and backend. And let's keep using rspec/jasmine tests for everything else. In this case I don't think that we need more than one cuke for the “mark read/unread” feature.